config/dev.exs

```diff
@@ -42,7 +44,8 @@ config :mongoose_push,
       endpoint: "localhost",
       mode: :dev,
       use_2197: true,
-      pool_size: 5
+      pool_size: 5,
+      tls_opts: []
```

Comment:
This option is here, as recently MongoosePush started validating APNS / FCM server certs by default. As for dev release we expect to be connecting to mocks, we need to disable the verification, as otherwise the release won't boot.

Reply:
Maybe we can add this option to the list here: https://github.com/esl/MongoosePush/commit/7983a853e7993ceca8205687c3397bb596aff1ea#diff-04c6e90faac2675aa89e2176d2eec7d8R194 ?